app/models/audio_file.rb

```diff
@@ -26,6 +26,7 @@ class AudioFile < BaseModel
   alias_attribute :duration, :length
 
   mount_uploader :file, AudioFileUploader, mount_on: :filename
+  skip_callback :commit, :after, :remove_file! # don't remove s3 file
```

Comment:
@kookster we actually already set `config.remove_previously_stored_files_after_update = false` in the carrierwave initializer.

But I confirmed that setting only applies to updates, not deletes.  Without the `skip_callback` the files will all be removed from S3, assuming CMS has permissions to do so.  (If it doesn't, you'll get a 500).